linode/manager
# fix: [M3-7170] - Create VPC Region and Label Validation

## Description 📝
Ensures that VPC region, label, and description validation error messages do not persist for VPC create flow.

## Preview 📷
https://github.com/linode/manager/assets/139280159/9da48bdc-ea37-472e-836b-350ecd8e2886

## How to test 🧪
1. **How to setup test environment?**
- With VPC flags on - on the dev environment or on the prod environment (with MSW): navigate to the Create VPC page.
2. **How to verify changes?**
- Without inputting anything, click on the Create VPC button. You should see errors pop up for the Region and VPC Label fields (as well as the subnet label field, but that's not part of this PR)
- Verify that when you select a region, the error disappears. 
- Verify that when you input a label, the error disappears.
- ** To test description error messages, you'll need to input a description > 255 characters 
- Verify that after initial scrolling to errors, the page does not continue to scroll if there are still errors on the page
